src/MongoDB.Driver.Legacy/ParallelScanArgs.cs

```diff
@@ -13,6 +13,7 @@
 * limitations under the License.
 */
 
+using System;
```

Comment:
Hm, was this line added accidentally?

Reply:
My bad, this is from when I thought I needed to edit this file as well. I'll make sure to delete that.